src/ResourceManager/KeyVault/Commands.KeyVault/Properties/Resources.resx

```diff
@@ -147,6 +147,9 @@ You can find the object ID using Azure Active Directory Module for Windows Power
   <data name="InvalidApplicationId" xml:space="preserve">
     <value>Invalid application Id.</value>
   </data>
+  <data name="InvalidObjectIdSyntax" xml:space="preserve">
+    <value>Cannot parse ObjectId into Guid.</value>
```

Comment:
As mentioned by @jkapil, there still seems to be an ongoing discussion on whether or not ADFS scenarios will use an SID format for their object IDs. Until a definitive decision has been made, we will defer explicitly checking for an SID format.